tamr_unify_client/project/attribute_mapping/resource.py

```diff
@@ -64,6 +64,11 @@ def resource_id(self):
         spliced = self.relative_id.split("attributeMappings/")[1]
         return spliced
 
+    def delete(self, url, client):
```

Comment:
This shouldn't need to take in url and client. It seems like right now attribute mapping does not have client and api path properties, which is an issue here. The attribute mapping `__init__` should take these in to fix this (api path can be optional, as it is the same as the mapping's relative ID--do this similarly to how base resource does)

Also needs a docstring